src/postTransformer.ts

```diff
@@ -68,6 +70,33 @@ function insertReferences(root: Root, references: ReferenceMapping): void {
   }
 }
 
+// to add this function due to classes under modules need to be cross reference
+export function insertClassReferenceForModule(flattenElements: Root[]) {
+  flattenElements.forEach(function (element) {
+    if (element.items[0].type === 'module') {
+      let references = element.references;
+      let children = element.items[0].children as string[];
+      children.forEach(function (child) {
+        let find = false;
```

Comment:
since you return when find, I think find is not needed.

Reply:
it is required.  return just jump out of the the current forEach.